brave/browser-laptop
# Move NoScript icon into URL bar. This fixes draggability dead area

## Test plan

1. Make sure the non-draggable area specified in #5792 is draggable
2. Make sure every button on the navigation bar is clickable
3. Open https://jsfiddle.net/
4. Block scripts from the top right lion icon
5. Make sure the noScript button on the navigation bar is clickable

## Description

Fixes https://github.com/brave/browser-laptop/issues/5792

Auditors: @bradleyrichter, @luixxiul


![screen shot 2017-04-11 at 11 51 08 am](https://cloud.githubusercontent.com/assets/4733304/24926914/1026bee2-1eb2-11e7-8457-663eecc0e3db.png)
